core/src/test/scala/org/apache/spark/SparkFunSuite.scala

```diff
@@ -72,3 +99,27 @@ abstract class SparkFunSuite
   }
 
 }
+
+object SparkFunSuite
+  extends Logging {
```

Comment:
move to previous line

Reply:
Object removed due to previous review items.